src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -50,6 +52,30 @@ public static int RunProcess (string name, string args, DataReceivedEventHandler
 			}
 		}
 
+		static string GetOSBinDirName ()
+		{
+			if (Path.DirectorySeparatorChar == '\\')
+				return "";
+			string os = null;
+			DataReceivedEventHandler output = (o, e) => {
+				if (string.IsNullOrWhiteSpace (e.Data))
+					return;
+				os = e.Data.Trim ();
+			};
+			DataReceivedEventHandler error = (o, e) => {};
+			int r = RunProcess ("uname", "-s", output, error);
```

Comment:
Yes, based on the `Path.DirectorySeparatorChar` check. That could be `OS.IsWindows` if you prefer.